rust/src/dataset/transaction.rs

```diff
@@ -391,10 +396,12 @@ impl TryFrom<&pb::Transaction> for Transaction {
             },
             Some(pb::transaction::Operation::Overwrite(pb::transaction::Overwrite {
                 fragments,
+                indices,
```

Comment:
In most overwrite scenarios it may not be correct to inherit the existing indices. e.g., if you completely overwrite the table with new data. Just want to make sure we don't accidentally inherit when we don't mean to.

Reply:
The way I was thinking about it was an "overwrite" is "completely replace the existing data with new data".  That new data might be "just data" or it might be "data + indices".  It just depends on the source of the new data.